diff --git a/tools/cmake/project.cmake b/tools/cmake/project.cmake
index e35e88399..7cd626120 100644
--- a/tools/cmake/project.cmake
+++ b/tools/cmake/project.cmake
@@ -391,6 +391,7 @@ macro(project project_name)
     add_custom_target(_project_elf_src DEPENDS "${project_elf_src}")
     add_executable(${project_elf} "${project_elf_src}")
     add_dependencies(${project_elf} _project_elf_src)
+    target_link_libraries(${project_elf} "-Wl,--start-group")
 
     if(test_components)
         target_link_libraries(${project_elf} "-Wl,--whole-archive")
