From 0439a9785dc9da8ad5f797b95bcaeb40e7c84b17 Mon Sep 17 00:00:00 2001
From: Carl Lundin <lundinc@amazon.com>
Date: Thu, 15 Oct 2020 10:23:51 -0700
Subject: [PATCH] Add basic to include path.

---
 CMakeLists.txt | 1 +
 1 file changed, 1 insertion(+)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index c64bc87..29a636e 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -23,6 +23,7 @@ set(COMPONENT_SRCS          "${COMPONENT_DIR}/cryptoauthlib/lib/hal/atca_hal.c"
 set(COMPONENT_INCLUDEDIRS   "${CRYPTOAUTHLIB_DIR}/"
                             "${CRYPTOAUTHLIB_DIR}/hal"
                             "${CRYPTOAUTHLIB_DIR}/../app/tng"
+                            "${CRYPTOAUTHLIB_DIR}/basic"
                             "port"
                             )
 
-- 
2.28.0

